internal/jira/client.go

```
@@ -0,0 +1,254 @@
+		components := []string{}
+		for _, component := range issue.Fields.Components {
+			if component != nil {
+				components = append(components, component.Name)
+			}
+		}
+		if len(components) > 0 {
+			ticket.Fields["components"] = components
+		}
+	}
+
+	if issue.Fields.Labels != nil && len(issue.Fields.Labels) > 0 {
```

golangci-lint (1.64.8)

S1009: should omit nil check; len() for nil slices is defined as zero

(gosimple)